src/docs/markdown/install.md

```diff
@@ -37,6 +37,7 @@ Our [official packages](https://github.com/caddyserver/dist) come only with the
 
 1. Obtain a Caddy binary:
 	- [from releases on GitHub](https://github.com/caddyserver/caddy/releases) (expand "Assets")
+		- Refer to [Verifying Asset Signatures](/docs/signature-verification) for how to verify the asset signature
```

Comment:
I'm not exactly sure what's the best way to link to the article from here, so I put up a sub-sub-bullet for now 🤷🏼‍♂️